cmd/operator/main.go

```diff
@@ -108,6 +108,8 @@ var (
 	// Parameters for the kubelet endpoints controller.
 	kubeletObject   string
 	kubeletSelector operator.LabelSelector
+
+	nodeAddressPriority string
```

Comment:
I'd recommend using a custom type instead of string (e.g. `type NodeAddresPriority string`) and implement the flag.Set interface for it. This way we can validate the flag value before starting everything.